### ISSUE TITLE
Content not matching the title

### ISSUE THREAD

**Issue by samusaran** [SharePoint/sp-dev-docs#2264]
The title of the article is "Deploying your SharePoint Add-ins", but it does not contains any guidelines to deploy the addins, only informations that helps solving a couple of specific problems.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9699e2fa-fd77-5147-aa9b-919c72242621
* Version Independent ID: 56dae12f-77d8-501e-6f70-fc4922e8fa97
* Content: [Deploying your SharePoint Add-ins](https://docs.microsoft.com/en-us/sharepoint/dev/solution-guidance/deploying-your-sharepoint-add-ins)
* Content Source: [docs/solution-guidance/deploying-your-sharepoint-add-ins.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/solution-guidance/deploying-your-sharepoint-add-ins.md)
* Product: **sharepoint**
* GitHub Login: @spdevdocs
* Microsoft Alias: **spdevdocs**

**Comment by andrewconnell** [comment]
Correct... it contains links to common things that come up with the deployment of SharePoint Add-ins. What specifically are you looking for / what's missing?

**Comment by samusaran** [comment]
I was developing a provider-hosted addin for the first time and I had to understand how to deploy the addin and then let Sharepoint Online know my addin. All Tasks I had to do were scattered around the documentation, while I expected a more specific section dedicated to how to configure Sharepoint and the Addin to let everything work. Some steps were not so clear and I took a couple of days to get everything working, while basically digging through the documentation.

I'm speaking as a developer that had literally no experience in Sharepoint ever, so from my point of view a better documentation really would had helper me to get everything to work faster.

**Comment by andrewconnell** [comment]
Appreciate the comment... however, a bit more context would help in understanding & addressing the challenge. Virtually everyone working on the docs is coming from a SharePoint background so having someone who doesn't have that background provide feedback helps... so thanks!

Can you share details around this:

> All Tasks I had to do were scattered around the documentation, while I expected a more specific section dedicated to how to configure Sharepoint and the Addin to let everything work. Some steps were not so clear and I took a couple of days to get everything working, while basically digging through the documentation.

What tasks? What pages did you find your answers, what was confusing? 

Specifics help... without them it's hard to know what to fix...

**Comment by samusaran** [comment]
I'm sorry if my answer is vague, but since my post here I left that company (so the project) and I don't really remember the details anymore, nor have the codebase that I can look to.

I remember we had issues about:

- Uploading the addin package to Sharepoint Online
- Registering the application (I remember I had to use an URL not available from anywhere)
- Troubleshoot authentication errors on the addin derived from wrong configuration on the application

Basically my issue with this capter of the documentation was that it really didn't covered what the title said. If I see an articled title "Deploying your SharePoint Add-ins" I suppose that I will find everything related to the Deploy of my addin.

In fact the title of this issue is about the "content not matching the title"; the issue could be with the missing content _or_ the misleading title, in relationship about what do you want to treat this.

**Comment by andrewconnell** [comment]
Thanks... I see how this could be a bit confusing. I'll add this to the backlog to get a bit more treatment.
